future_challenges_for_RSEng.tex

```diff
@@ -278,14 +279,39 @@ \subsection{Definition and Evolution of RSEs}
 % European quantum computing: https://digital-strategy.ec.europa.eu/en/policies/quantum
 \end{FramedParagraphWithFootnotes}
 
+\begin{FramedParagraphWithFootnotes}
```

Comment:
I think that is a good point, also if the funding isn't solely project specific an RSE might have some spare capacity to deal with legacy codes